NorthstarDLL/mods/autodownload/moddownloader.cpp

```diff
@@ -0,0 +1,525 @@
+#include "moddownloader.h"
+#include <rapidjson/fwd.h>
+#include <mz_strm_mem.h>
+#include <mz.h>
+#include <mz_strm.h>
+#include <mz_zip.h>
+#include <mz_compat.h>
+#include <thread>
+#include <future>
+#include <bcrypt.h>
+#include <winternl.h>
+#include <fstream>
+
+ModDownloader* g_pModDownloader;
+
+ModDownloader::ModDownloader()
+{
+	spdlog::info("Mod downloaded initialized");
+	modState = {};
```

Comment:
This doesn't rly need to be initialised in the constructor

Reply:
Addressed in 4f2394e4bef0650c56c9b790dffc497669341ee0